src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -507,7 +509,7 @@ public ImmutableArray<CodeAction> GetCodeActionsForFixes(Document document, Immu
 
                 codeActionsBuilder.AddIfNotNull(codeAction);
 
-                if (limitResults && codeActionsBuilder.Count >= MaxResults)
+                if (codeActionsBuilder.Count >= maxResults)
```

Comment:
can you get rid of the constant at the top of the file now?

Reply:
Looks like I'll need to pass it in to some other methods in order to remove the constant.